apps/fabric-website/gulpfile.js

```diff
@@ -33,62 +26,29 @@ build.tslint.setConfig({
   lintConfig: { rules },
   sourceMatch: ['src/**/*.ts', 'src/**/*.tsx', '!src/**/*.scss.tsx']
 });
-
+// TODO: remove this! There are a number of lint errors to fix.
```

Comment:
This is actually just moving the line up from the bottom to live next to the actual mixed in rules that should be on for the website, but aren't, for one reason or another.

@battletoilet fyi